### ISSUE TITLE
URL string not reflected in the api sub-title properly when given as api value annotation

### ISSUE THREAD

**Issue by shashi0905** [swagger-api/swagger-ui#1655]
With swagger annotation @Api (value=" "), when I try to insert an URL string, swagger UI skips all the forward slashes (/) while display.

Annotation being used : 

@Api(value="shoplocator- http://api.meteor.com/v2/shops", description="some description...")
public class SomeClass {  

What I'm getting at swagger UI :
shoplocator- http:api.meteor.comv2shops

What I'm expecting :
shoplocator- http://api.meteor.com/v2/shops

I tried using escaspe sequences and other ways to handle special characters, but nothing worked, like : shoplocator- http:\//api.meteor.com\/v2\/shops or shoplocator- http:\&#47;\&#47;api.meteor.com\&#47;v2\&#47;shops
But nothing worked. 
How can I get the proper URL string in the api sub title value?

**Comment by webron** [comment]
This is actually an issue with swagger-core, so you'd need to reopen there if you want us to look into it. One question I'd raise is why are you trying to put a url in the value? It's not that it is not allowed, but it seems like you may try doing different than what the value is actually meant to do.

**Comment by shashi0905** [comment]
Hey, thanks for your response.

I understand your question, actually it's a requirement from the client. And it was not about displaying the URL string in the value, at the first place. Initially our requirement was to display a URL link below (just after) the response definitions. 
I was trying to include external-docs through swagger annotations and code configuration. But it didn't reflected on swagger UI. 
Also, $ref attribute (to read the error definitions from a separate json file) also didn't work properly as expected. I opened an issue related to this problem at https://github.com/swagger-api/swagger-ui/issues/1614 but got no response. 

When we didn't get a solution for the above problem, we decided, for the time being, to move the proposed URL link as plain text in the sub titles section. Which, in turn, again didn't work !!
I'll see if I get some suggestion at swagger-core, otherwise we might keep the URL string at some other place.
